template.yaml

```diff
@@ -141,6 +141,21 @@ Resources:
       VersioningConfiguration:
         Status: Suspended
 
+  # Bucket for storing API responses that are too large to pass through Lambda
+  LargeApiResponsesBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !If
+        - WithSuffix
+        - !Sub 'large-api-responses-${AWS::AccountId}-${Suffix}'
+        - !Sub 'large-api-responses-${AWS::AccountId}'
```

Comment:
Suggestions for a better name? Harder to change this later

Reply:
The name is ok, chatgpt comes with following suggestions :)

api-payload-bucket
api-result-storage-bucket
api-response-archive-bucket